tests/bsim/bluetooth/ll/advx/src/main.c

```diff
@@ -675,28 +679,32 @@ static void test_advx_main(void)
 
 	k_sleep(K_MSEC(1000));
 
-	printk("Add to resolving list...");
-	bt_addr_le_t peer_id_addr = {
-		.type = BT_ADDR_LE_RANDOM,
-		.a = {
-			.val = {0xc6, 0xc7, 0xc8, 0xc9, 0xc1, 0xcb}
-		}
-	};
-	uint8_t pirk[16] = {0x00, };
-	uint8_t lirk[16] = {0x01, };
+	if (IS_ENABLED(CONFIG_BT_CTLR_PRIVACY)) {
+		printk("Add to resolving list...");
+		bt_addr_le_t peer_id_addr = {
+			.type = BT_ADDR_LE_RANDOM,
+			.a = {
+				.val = {0xc6, 0xc7, 0xc8, 0xc9, 0xc1, 0xcb}
+			}
+		};
+		uint8_t pirk[16] = {0xAB, 0xBA, 0xAB, 0xBA, 0xAB, 0xBA, 0xAB, 0xBA,
+				    0xAB, 0xBA, 0xAB, 0xBA, 0xAB, 0xBA, 0xAB, 0xBA};
+		uint8_t lirk[16] = {0x12, 0x21, 0x12, 0x21, 0x12, 0x21, 0x12, 0x21,
+				    0x12, 0x21, 0x12, 0x21, 0x12, 0x21, 0x12, 0x21};
 
-	err = ll_rl_add(&peer_id_addr, pirk, lirk);
-	if (err) {
-		goto exit;
-	}
-	printk("success.\n");
+		err = ll_rl_add(&peer_id_addr, pirk, lirk);
+		if (err) {
```

Comment:
It's unclear to me whether, when moving/indenting existing code, whether the code should still be updated to follow the current guidelines or not :|

E.g. this should be
```suggestion
		if (err != 0) {
```

But not sure if that's something we want to do when just moving code